src/css/jwplayer/states.less

```diff
@@ -20,7 +20,6 @@ body .jwplayer.jw-state-error {
     font-size: @default-em-size;
     display: table;
     opacity: 1;
-    overflow: hidden;
```

Comment:
Is `overflow: hidden` already applied elsewhere? A developer might be resizing the player in the error state specifically to hide the error from users.

Reply:
Yes, in the `info-overlay` div. It doesn't make sense to have `overflow:hidden` on the outermost container because nothing inside can overflow.

Reply:
OK. We should have that on the player contain already anyway.